runner/master/run.sh

```diff
@@ -741,6 +785,13 @@ echo "==========================================================================
 docker exec -t "${WEBSERVER}" \
   chown -R "${UID}:${GROUPS[0]}" /shared
 
+# Delete the temporary folder.
+if [ -v PLUGINSTMPDIR ]
+then
+  echo "== Removing temporary folder."
+  rm -rf ${PLUGINSTMPDIR}
```

Comment:
If we use the proposal above of specifyign a default PLUGINDIR, then we don't need this bit at all. It will be cleaned up by Jenkins as part of the Workspace cleanup.